libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -440,11 +440,8 @@ AP_GPS_SBF::parse(uint8_t temp)
 
 static bool is_DNU(double value)
 {
-    constexpr double DNU = -2e-10f;
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wfloat-equal" // suppress -Wfloat-equal as it's false positive when testing for DNU values
-    return value != DNU;
-#pragma GCC diagnostic pop
+    constexpr double DNU = -2e10;
+    return is_equal(value, DNU);
```

Comment:
Please just change the one line - we want string bitwise-equivalence for this check, rather than the epsilon-based thing that `is_equal` uses.

@Tory9 FYI